pkg/app/app.go

```diff
@@ -190,9 +190,9 @@ func (app *App) setupRepo() (bool, error) {
 						return true, nil
 					}
 				}
-				return false, err
 			}
 
+			fmt.Println("Must open lazygit in a git repository.", app.Tr.NoRecentRepositories)
```

Comment:
I would internationalise the first part here too. In fact I'd have it as a single string in english.go